web-app/src/app/screens/FeedSubmissionFAQ.tsx

```diff
@@ -46,6 +46,36 @@ export default function FeedSubmissionFAQ(): React.ReactElement {
           borderRadius: '6px 6px 0px 0px',
         }}
       >
+        <Accordion sx={accordionStyle}>
+          <AccordionSummary
+            aria-controls='panel1-content'
+            id='panel1-header'
+            expandIcon={<ExpandMoreIcon />}
+          >
+            <Typography sx={{ fontWeight: 'bold' }}>
+              Can I contribute GBFS feeds?
+            </Typography>
+          </AccordionSummary>
+          <AccordionDetails>
+            <Typography>
+              If you want to add GBFS feeds to the Mobility Database, please
+              contribute to
+              <Button
+                variant='text'
+                className='inline'
+                href={
+                  'https://github.com/MobilityData/gbfs?tab=readme-ov-file#systems-catalog---systems-implementing-gbfs'
+                }
+                rel='noreferrer'
+                target='_blank'
+                endIcon={<OpenInNewIcon />}
+              >
+                the GBFS systems catalog.
```

Comment:
@emmambd Suggestion to use the name systems.csv.

```suggestion
                the GBFS systems.csv catalog.
```